package.json

```diff
@@ -6,7 +6,7 @@
   "contributors": [],
   "license": "MIT",
   "scripts": {
-    "build": "shx rm -rf dist && webpack && pkg dist/spk.js && shx mv spk-{linux,macos,win.exe} dist",
+    "build": "shx rm -rf dist && webpack && pkg -t node12-linux-x64,node12-macos-x64,node12-win-x64 dist/spk.js && shx mv spk-{linux,macos,win.exe} dist",
```

Comment:
@andrebriggs note this. The `build` script now always try's to build on a node 12 binary instead of whatever is on the host machine.

Reply:
@evanlouie That should be fine. the Node 12 build produces the build artifact.